packages/connect/src/data/models.ts

```diff
@@ -17,4 +17,8 @@ export const models = {
             '5': 'Bitcoin Orange',
         },
     },
+    T3T1: {
+        name: 'Trezor TURBO',
```

Comment:
:laughing:

Reply:
psst, the name is secret

Reply:
@mroz22 no it's not 😉

Reply:
We can use the full name or a shortcut, please adjust the code so we will not forget anything. Thanks! :)